applications/ray/kuberay-tpu-webhook/main.go

```diff
@@ -242,6 +256,7 @@ func validateRayCluster(admissionReview *admissionv1.AdmissionReview) (*admissio
 	status := "Success"
 	message := ""
 	clusterName := raycluster.Name
+	klog.Infof("Validating RayCluster %s", clusterName)
```

Comment:
log namespaced name 
```
klog.Infof("Validating RayCluster %s/%s", raycluster.Namespace, raycluster.Name)
```

Reply:
Done, I changed it to structured logging but used this format to add in the namespaced name.